22-Natural-Language-Processing/index.html

```diff
@@ -5,18 +5,34 @@
   <link rel="stylesheet" href="../styles.css">
   <script src="http://code.jquery.com/jquery-1.12.4.js"></script>
   <script type="text/javascript" src="../main.js"></script>
+  <script type="text/javascript" src="../22-Natural-Language-Processing/nGramModels.js"></script>
 </head>
 <body>
 
   <div class="row">
     <div class="col-sm-6 col-md-offset-3" id="content">
       <h1>Natural Language Processing</h1>
-      <p>TODO</p>
+      <h3>Table of contents</h3>
+      <ul class="content-table">
+        <li><a href="#n-gram-character-models">N-gram character models</a></li>
+      </ul>
 
-      
+      <h2 id="n-gram-character-models">N-gram character models</h2><br>
+
+      <form id="form">
+        <div class="form-group">
+          <label >Enter a phrase</label>
+          <input type="text" class="form-control" name="firstname" id="phrase"/>
+        </div>
+        <div class="form-group">
+          <button type="submit" class="submit btn btn-success">Submit</button>
+        </div>
+      </form>
+      <span><h5>Character Level Unigrams :-</h5></span><div id="character-level-unigram"></div><br>
```

Comment:
A `span` is an “inline element” and an `h5` is a “block element”. Normally a block element can contain inline elements, but inline elements should not contain block elements. I think you can remove the `span` here